beanie/odm/utils/init.py

```diff
@@ -1,6 +1,8 @@
 import asyncio
 import sys
 
+from typing_extensions import Sequence, get_args, get_origin
+
```

Comment:
I'm not sure if this is the best idea.

`typing_extensions` is supposedly only installed if Python version is less than 3.11, as per the pyproject.
![image](https://github.com/user-attachments/assets/795c193f-b49d-4c9b-8f67-3642efa94b9e)

Wouldn't this fail for those with Python version >= 3.11?

The only reason why the tests are not failing for Python >= 3.11 is because the typing_extensions is getting installed as part of fastapi (which is defined as an optional dependency for testing).

Reply:
What's the benefit of a sequence?

Reply:
@adeelsohailahmed now this is uniform for everything.... it will simplify import logic everywhere

@mg3146 just typing... anything that can be iterated over by for should be valid input